src/components/fetchData.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import Quotes from './quotes';
+
+export default function FetchData() {
+  const [quote, setQuote] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const apiKey = 'lGdmxaApU87yDaz1n5zNbS6zBKQ4lFVdikwORfjk';
+  const category = 'computers';
+
+  useEffect(() => {
+    setIsLoading(true);
+    try {
+      fetch(`https://api.api-ninjas.com/v1/quotes?category=${category}`, {
+        headers: {
+          'X-Api-Key': apiKey,
+        },
+      })
+        .then((response) => response.json())
+        .then((json) => setQuote(json[0]))
+        .then(() => setIsLoading(false));
```

Comment:
- [ ][OPTIONAL] you can actually set your loading state to false in the finally block, you can read more on promises in Js